_data/simple-icons.json

```diff
@@ -6573,6 +6573,11 @@
             "hex": "353E58",
             "source": "https://observablehq.com/"
         },
+        {
+            "title": "Obsidian",
+            "hex": "483699",
```

Comment:
Everything looks good except for the hex color here. I am not seeing this color anywhere on their website. I would go with `#7f6df2` which is under the CSS variable `--text-accent` on their website.

Reply:
Oh, nevermind I found it. It's the CSS variable `--interactive-accent`.